server/homebrew.api.js

```diff
@@ -6,13 +7,57 @@ const GoogleActions = require('./googleActions.js');
 const Markdown = require('../shared/naturalcrit/markdown.js');
 const yaml = require('js-yaml');
 const asyncHandler = require('express-async-handler');
+const { nanoid } = require('nanoid');
 
 // const getTopBrews = (cb) => {
 // 	HomebrewModel.find().sort({ views: -1 }).limit(5).exec(function(err, brews) {
 // 		cb(brews);
 // 	});
 // };
 
+const getBrew = (accessType)=>{
```

Comment:
I don't think I have any more major changes to add (still need to do more testing to make sure it's not broken anywhere). Only thing is I think these longer functions could benefit from some comments marking out each of the major steps since there's a lot of checking `googleId`, which may not be obvious. For instance, comments for:

1) Parse Brew ID
2) Search HB database for Brew (or Stub)
3) Search GDrive for Brew if needed

Reply:
Yeah adding comments is a great plan, I'll do that!

Reply:
Done!